bisys/src/main/kotlin/no/nav/familie/eksterne/kontrakter/bisys/BarnetilsynBisysResponse.kt

```diff
@@ -4,7 +4,7 @@ import java.time.LocalDate
 
 class BarnetilsynBisysResponse(
     val gjeldendeBarnIdent: String,
```

Comment:
Hvordan burde denne se ut ?

Reply:
gjeldendeBarnIdent? Tenker du på noe spesielt?

Reply:
Ja, vi får én ident for et barn som vi sender med her ?